src/main/java/com/rapid7/client/dcerpc/mslsad/LocalSecurityAuthorityService.java

```diff
@@ -99,6 +107,16 @@ public void closePolicyHandle(ContextHandle handle) throws IOException {
         transport.call(closeRequest);
     }
 
+    public LsarLookupNamesResponse lookupNames(ContextHandle policyHandle, String... names)
```

Comment:
Should we expose the response implementation from the service? Can we get a list of the struct from inside the response?

Reply:
Ah, if we need both the translatedSids and the domain list we can just keep it as is.

Reply:
I had a discussion with @tosmun-r7 about this. The resulting struct is complex enough to leave as is and let the top level service deal with it.